tests/aequilibrae/project/test_create_example.py

```diff
@@ -1,17 +1,20 @@
 from os.path import join
-from tempfile import gettempdir
-from uuid import uuid4
-from unittest import TestCase
 from aequilibrae.utils.create_example import create_example
 from aequilibrae import Parameters
+import pytest
 
 
-class Test(TestCase):
-    def test_create_example(self):
-        par = Parameters()._default
-        for p in ["nauru", "sioux_falls"]:
-            pth = join(gettempdir(), f"proj_example_{p}_{uuid4().hex}")
-            proj = create_example(pth, p)
-            parproj = proj.parameters
-            self.assertEqual(parproj.keys(), par.keys(), f"Wrong parameter keys for {p} example")
-            proj.close()
+class TestCreateExample:
+    # We use fixture parametrization here to create the projects so that we can do proper teardown
+    # of the project, even when the test fails
+    # see also: https://docs.pytest.org/en/6.2.x/fixture.html#fixture-parametrize
+    @pytest.fixture(params=["nauru", "sioux_falls"])
```

Comment:
I feel this is nice and easy to understand - and it doesn't seem to use any of the helper fixtures from conftest.py right?

Reply:
Nope, it needs to test `create_example` so using the helper fixtures would defeat the purpose ^^